frame/nomination-pools/src/lib.rs

```diff
@@ -1345,8 +1465,11 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Withdraw unbonded funds for the `target` delegator. Under certain conditions,
-		/// this call can be dispatched permissionlessly (i.e. by any account).
+		/// Withdraw unbonded funds who's unbonding period has passed for the `target` delegator. If
+		/// no bond can be unbonded, an error is returned.
+		///
+		/// Under certain conditions, this call can be dispatched permissionlessly (i.e. by any
+		/// account).
 		///
 		/// # Conditions for a permissionless dispatch
```

Comment:
```suggestion
		/// ## Conditions for a permissionless dispatch
```

Reply:
Same below

Reply:
yeah actually undid myself here for now. but just to be nitpicky and elaborate my reason, `#` is for main title, while we want something more like a sub-section here.

Reply:
b1ee72e83c859fa0539caa0cf0b249ca59b8282b